libcxx/utils/libcxx/test/features.py

```diff
@@ -500,6 +501,43 @@ def _getAndroidDeviceApi(cfg):
     ),
 ]
 
+
+# Detect whether dbx is on the system.
```

Comment:
Please explain what DBX is, I guess the platform debugger of AIX. LLDB and GDB are well known debuggers so there only the name suffices.

Reply:
Not just AIX: https://en.wikipedia.org/wiki/Dbx_(debugger)